Magento2/ruleset.xml

```diff
@@ -285,6 +285,11 @@
         <severity>8</severity>
         <type>warning</type>
     </rule>
+    <rule ref="Magento2.Legacy.ObsoleteMenu">
+        <include-pattern>*\/widget.xml$</include-pattern>
```

Comment:
This need to be done in widget.xml files? Shouldn't this need to be done in config.xml files?